playbooks/roles/nginx-proxy/templates/nginx-proxy.j2

```diff
@@ -14,7 +14,7 @@ server {
     {{ NGINX_PROXY_EXTRA_CONFIG }}
 
     set $expected_origin false;
-    if ($http_origin ~ {{ NGINX_PROXY_ORIGIN_REGEX }}) {
+    if (NGINX_PROXY_ORIGIN_REGEX and $http_origin ~ {{ NGINX_PROXY_ORIGIN_REGEX }}) {
```

Comment:
@kaustavb12 Isn't this still going to fail if the variable is undefined?

I'm guessing a safer approach is to set a default for `NGINX_PROXY_ORIGIN_REGEX` to `null` `nginx-proxy/defaults/main.yml` and then wrap the entire `if ($http_origin ~ {{ NGINX_PROXY_ORIGIN_REGEX }}) { ...` block into a jinja if `{% if NGINX_PROXY_ORIGIN_REGEX %} ... {% endif %}`.

There's another variable which was introduced in https://github.com/open-craft/ansible-playbooks/pull/269 and which I think should also have a default in `nginx-proxy/defaults/main.yml`. I think `[413]` would be a good default.

Reply:
@mtyaka You are absolutely right .. in a hurry I completely overlooked the fact that it is jija2 format and that the variable is actually undefined and not null. Thanks a lot for pointing this out.
I have also added the default for the other variable NGINX_PROXY_CORS_HEADERS_FOR_RESPONSE_CODES